modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cdc.kafka;
+
+import java.util.Collection;
+import java.util.Map;
+import org.apache.ignite.internal.processors.cache.IgniteInternalCache;
+
+/**
+ * Configuration of {@link KafkaToIgniteCdcStreamer} application.
+ *
+ * @see KafkaToIgniteCdcStreamer
+ * @see KafkaToIgniteLoader
+ */
+public class KafkaToIgniteCdcStreamerConfiguration {
+    /** Default {@link #kafkaParts} value. */
+    public static final int DFLT_PARTS = 16;
+
+    /** Default {@link #topic} value. */
+    public static final String DFLT_TOPIC = "ignite";
+
+    /** Default {@link #maxBatchSize} value. */
+    public static final int DFLT_MAX_BATCH_SIZE = 1024;
+
+    /** {@link Applier} thread count. */
+    private int threadCnt = DFLT_PARTS;
+
+    /** Topic name. */
+    private String topic = DFLT_TOPIC;
+
+    /** Kafka partitions count. */
+    private int kafkaParts = DFLT_PARTS;
```

Comment:
> Do we really need 2 params with the same defaults?

Yes.

Reply:
> Seems that having kafkaParts is enough and some topics per thread may be required, instead of partPerApplier calculation?

Different servers have a different cores available for different loads.
I don't think we should do some implicit calculation here - the user knows better how many threads he wants to create for Streamer.

Reply:
The current logic is not clear to me.
Having `threadCnt` and `kafkaParts` simultaneously with subsequent calculation `partPerApplier = kafkaParts / threadCnt` may set N partitions to some Appliers, but N+1 to other, that's not what user may expect.
We must simplify this to `kafkaParts` and `partPerApplier`.